[2024-06-15T21:07:28.062+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-15T21:07:28.093+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ran_info_api.format_response manual__2024-06-15T21:05:36.820154+00:00 [queued]>
[2024-06-15T21:07:28.108+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ran_info_api.format_response manual__2024-06-15T21:05:36.820154+00:00 [queued]>
[2024-06-15T21:07:28.109+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 6
[2024-06-15T21:07:28.341+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): format_response> on 2024-06-15 21:05:36.820154+00:00
[2024-06-15T21:07:28.353+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=7864) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-06-15T21:07:28.356+0000] {standard_task_runner.py:63} INFO - Started process 7875 to run task
[2024-06-15T21:07:28.353+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'ran_info_api', 'format_response', 'manual__2024-06-15T21:05:36.820154+00:00', '--job-id', '83', '--raw', '--subdir', 'DAGS_FOLDER/rand_request_api.py', '--cfg-path', '/tmp/tmpw5fkdwcd']
[2024-06-15T21:07:28.357+0000] {standard_task_runner.py:91} INFO - Job 83: Subtask format_response
[2024-06-15T21:07:28.764+0000] {task_command.py:426} INFO - Running <TaskInstance: ran_info_api.format_response manual__2024-06-15T21:05:36.820154+00:00 [running]> on host d4d3ae2e009b
[2024-06-15T21:07:28.972+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='protectorate' AIRFLOW_CTX_DAG_ID='ran_info_api' AIRFLOW_CTX_TASK_ID='format_response' AIRFLOW_CTX_EXECUTION_DATE='2024-06-15T21:05:36.820154+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-15T21:05:36.820154+00:00'
[2024-06-15T21:07:28.977+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-15T21:07:28.999+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-15T21:07:29.000+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/rand_request_api.py", line 17, in format_response
    result = res['results'][0]
             ~~~^^^^^^^^^^^
TypeError: string indices must be integers, not 'str'
[2024-06-15T21:07:29.016+0000] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=ran_info_api, task_id=format_response, run_id=manual__2024-06-15T21:05:36.820154+00:00, execution_date=20240615T210536, start_date=20240615T210728, end_date=20240615T210729
[2024-06-15T21:07:29.091+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 83 for task format_response (string indices must be integers, not 'str'; 7875)
[2024-06-15T21:07:29.139+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-06-15T21:07:29.172+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-15T21:07:29.283+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
